app/constant.js

```diff
@@ -0,0 +1,60 @@
+//All ERROR MESSAGE LABEL
+export const BANNER_ERROR_MSG = "Banner list get Api Failed!";
+export const CATEGORIES_ERROR_MSG = "Product list get Api Failed!";
+export const ERROR = "error";
+export const PRODUCTS_ERROR_MSG = "Products list get Api Failed!";
+
+
+
+
+// Login Signup page label and ERROR Messages
+export const LOGIN_SIGNUP_MSG = {
+  LOGIN_TEXT: "LogIn",
+  LOGIN_DESCRIPTION: "Get access to your Orders, Wishlists and Recommendations.",
+  EMAIL_REQUIREED: "Email is required",
+  PASSWORD_REQUIREED: "Password is required",
+  SIGNUP_TEXT: "Signup",
+  SIGNUP_DESCRIPTION: "We do not share your personal details with anyone.",
+  EMAIL_REQUIREED: "Email is required",
+  PASSWORD_REQUIREED: "Password is required",
+  FIRSTNAME_REQUIRED: "First Name is required",
+  FIRSTNAME_MINCHAR: "First Name is too short - should be 3 chars minimum.",
+  LASTNAME_REQUIRED: "Last Name is required",
+  LASTNAME_MINCHAR: "Last Name is too short - should be 1 chars minimum.",
+  PASSWORD_MINCHAR: "Password is too short - should be 6 chars minimum.",
+  PASSWORD_NUMBER: "Password must contain a number.",
+  CONFIRM_PASSWORD_REQUIRED: "Confirm Password is required",
+  CONFIRM_PASSWORD_MINCHAR: "Confirm Password is too short - should be 6 chars minimum.",
+  CONFIRM_PASSWORD_NUMBER: "Confirm Password must contain a number.",
+  CONFIRM_PASSWORD_MATCH: "Passwords must match",
+}
+
+// Cart page label and messages
+export const CART_LABEL_MESSAGE = {
+  CHEAPER_MSG: "You won't find it cheaper anywhere",
+  PROMO_CODE_TEXT: "Promo code can be applied on payment page",
+  CHECKOUT_BTN_TEXT: "Proceed to Checkout",
+  EMPTY_CART_MSG: "No items in your cart",
+  FAVOURITE_ITEMS_TEXT: "Your favourite items are just a click away",
+  START_SHOPPING_TEXT: "Start Shopping"
+}
+export const HEADER_LABEL_MESSAGE = {
+  HEADER_LINK1: "Home",
+  HEADER_LINK2: "Product",
+  HEADER_LINK3: "SignIn",
+  HEADER_LINK4: "Register",
+}
+// Product page label and messages
+export const PRODUCT_LABEL_MSG = {
+  CURRENCY_TEXT: "MRP Rs.",
+  BUTTON_TEXT: "Buy Now",
+  BUTTON_TEXT_WITH_CURRENCY: "Buy Now @ Rs."
+}
+
+// Footer section label and messages
+
+export const FOOTER_TEXT = "Copyright &copy; 2011-2018 Sabka Bazar Grocery Supplies Pvt Ltd.";
+
```

Comment:
if possible try to inject current year dynamically. 2011-2021

Reply:
Done.